config.yaml

```diff
@@ -27,6 +27,41 @@ source_config:
 # remove to disable S3 uploads
 s3_bucket: fcos-builds
 
+default_artifacts:
+  aarch_64:
```

Comment:
Shouldn't this be `aarch64`?

Reply:
yep. I'll fix it in my open PR.

Reply:
added a commit to https://github.com/coreos/fedora-coreos-pipeline/pull/645